test/overlay-position.html

```diff
@@ -6,6 +6,7 @@
   <title>vaadin-combo-box resizing tests</title>
 
   <script src='../../web-component-tester/browser.js'></script>
+  <script src='../../webcomponentsjs/webcomponents-lite.js'></script>
```

Comment:
@platosha I know that this is really in "internal" PR, but I was wondering why y'all are using `webcomponents-lite.js` instead of `webcomponents-loader.js`. The later is the recommended way for apps to load polyfills in the app itself. This could affect edge issues with respect to native vs polyfilled implementations on older browsers. Just a thought.

Reply:
Good question. I’ve just used the same that was included in the `test/index.html` for now. 

That’s the leftover from Polymer v1 days here, I think, so we should probably change this.

Although I don’t think this is urgent. AFAIK, the polyfills disable themselves in case if native support is present. So the only practical side-effect of `-lite` is a few Kbytes of dead code in the tests in that case.

Reply:
@platosha Thanks. Didn't remember that the polys auto-disable. For the life of me I can't remember if that has always been the case, or if that's a v2 enhancement.